src/scf-release/src/acceptance-tests-brain/test-scripts/011_nfspersi_test.rb

```diff
@@ -33,6 +34,15 @@
         run "cf delete-service -f #{VOLUME_NAME}"
         run "cf disable-service-access persi-nfs"
 
+        # Get logs and more information about the test server, why it
```

Comment:
I don't think we should merge with these changes. Maybe mark the PR as WIP and have it to run on the various CIs from the spitted images and helm charts from this PR?